addons/mail/static/src/discuss/call/common/peer_to_peer.js

```diff
@@ -818,6 +834,9 @@ export class PeerToPeer extends EventTarget {
                 return;
             }
             peer.isBuildingOffer = false;
```

Comment:
Shouldn't this be reset in "disconnect" too?

Reply:
It's a property of Peer and we clear pears on disconnect, they are not recycled